From: Gianfranco Costamagna <locutusofborg@debian.org>
Date: Wed, 27 Sep 2017 00:29:24 +0200
Subject: Remove libdir from sdl2-config

This makes the -dev package multiarch co-installable, allowing compilation
of SDL programs for more than one architecture on the same system.

This is not an upstreamable change, since it relies on a Debian-specific
assumption (that we are installing the library into a directory on the
linker's default search path).

[smcv: Only patch sdl2-config.in, not sdl2-config.cmake.in. It's OK for
sdl2-config.cmake to be architecture-dependent, because it gets installed
in ${libdir} anyway.]

Forwarded: not-needed
Last-Update: 2022-06-13
---
 sdl2-config.in | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/sdl2-config.in
+++ b/sdl2-config.in
@@ -7,7 +7,6 @@
 prefix=$(cd -P -- "$bindir/@bin_prefix_relpath@" && printf '%s\n' "$(pwd -P)")
 exec_prefix=@exec_prefix@
 exec_prefix_set=no
-libdir=@libdir@
 
 @ENABLE_STATIC_FALSE@usage="\
 @ENABLE_STATIC_FALSE@Usage: $0 [--prefix[=DIR]] [--exec-prefix[=DIR]] [--version] [--cflags] [--libs]"
@@ -49,12 +48,12 @@
       echo -I@includedir@/SDL2 @SDL_CFLAGS@
       ;;
 @ENABLE_SHARED_TRUE@    --libs)
-@ENABLE_SHARED_TRUE@      echo -L@libdir@ @SDL_RLD_FLAGS@ @SDL_LIBS@
+@ENABLE_SHARED_TRUE@      echo @SDL_RLD_FLAGS@ @SDL_LIBS@
 @ENABLE_SHARED_TRUE@      ;;
 @ENABLE_STATIC_TRUE@@ENABLE_SHARED_TRUE@    --static-libs)
 @ENABLE_STATIC_TRUE@@ENABLE_SHARED_FALSE@    --libs|--static-libs)
 @ENABLE_STATIC_TRUE@      sdl_static_libs=$(echo "@SDL_LIBS@ @SDL_STATIC_LIBS@" | sed -E "s#-lSDL2[ $]#$libdir/libSDL2.a #g")
-@ENABLE_STATIC_TRUE@      echo -L@libdir@ $sdl_static_libs
+@ENABLE_STATIC_TRUE@      echo $sdl_static_libs
 @ENABLE_STATIC_TRUE@      ;;
     *)
       echo "${usage}" 1>&2
